test/fixtures/condition/condition-with.htl

```diff
@@ -0,0 +1 @@
+<sly data-sly-test="${ author }"><h2>By ${ firstName } ${ lastName }</h2></sly>
```

Comment:
It this correct? In `hbs` the name fields are nested within the `with` block, so without changing scope in `htl`, shouldn't it be `author.firstName` ?